util/exrp/env/env.go

```diff
@@ -0,0 +1,12 @@
+package env
+
+import (
+	"github.com/Masterminds/sprig"
+	"github.com/doublerebel/bellows"
+)
+
+func New(m map[string]interface{}) map[string]interface{} {
```

Comment:
minor- 
Instead. of `New` can we have a better name `SetExpressionEnv`

Reply:
this is not a setter, so should probably not start with `Set` but how about `WithDefaults`